packages/react/package.json

```diff
@@ -68,7 +69,7 @@
     "classnames": "2.3.1",
     "deepmerge": "4.2.2",
     "lodash": "4.17.21",
-    "mapbox-gl": "npm:empty-npm-package@1.0.0",
+    "mapbox-gl": "file:lib/mapbox-gl",
```

Comment:
Does that mean `file` alias is more widely accepted than `npm` alias?

Reply:
We've had some problems when using hosted apps with the `npm` prefix (like npm pretty much won't resolve the package as aliased before). I've verified that the `file:` prefix works in a hosted Amplify Console application.